tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4051,6 +4051,7 @@ declare namespace ts {
         newLine?: NewLineKind;
         omitTrailingSemicolon?: boolean;
         noEmitHelpers?: boolean;
+        indentation?: number | "\t"; /** number of spaces per indentation level or single tab per level */
```

Comment:
See previous; this shows how the comment is misplaced.

Reply:
baselines have been rebuilt after the comment was moved